Cargo.toml

```diff
@@ -27,8 +27,10 @@ metriken = "0.6.0"
 metriken-exposition = { version = "0.5.0", features = ["json", "parquet-conversion"] }
 mio = "0.8.8"
 momento = "0.32.1"
-pelikan-net = "0.1.0"
+pelikan-net = { git = "https://github.com/brayniac/pelikan", default-features = false }
```

Comment:
This has been published, hasn't it?

Reply:
Good catch. This was leftover from testing. Fixed.